code/load/scenarios.R

```diff
@@ -143,5 +192,97 @@ _** If a second booster is already offered to 60+ (Greece, Netherlands), a third
     </td>
   </tr>
 </table>"
+  ),
+  # -------------------------------------------------------------------------
+  # Round 3
+  "round-3" = list("round" = 3,
+                   # Short text label for scenarios in format:
+                   #  "Policy scenario / Epidemiological scenario"
+                   "scenario_labels" = c(
+                     "A" = "A. No vaccination, optimistic variants",
+                     "B" = "B. Annual vaccination, optimistic variants",
+                     "C" = "C. Biannual vaccination, optimistic variants",
+                     "D" = "D. No vaccination, pessimistic variants",
+                     "E" = "E. Annual booster campaign, pessimistic variants",
+                     "F" = "F. Biannual booster campaign, pessimistic variants"
+                     ),
+                   # Key dates
+                   "origin_date" = as.Date("2022-09-11"),
+                   "submission_window_end" = as.Date("2022-09-16"),
+                   # Short scenario guide (for website + figure captions)
+                   "scenario_caption" = "Scenarios: New vaccination (none/annual/biannual); \n New variants (optimistic/pessimistic)",
+                   # HTML format scenario table
+                   "table" = "<table> <tr>
+      <td>
+        Notes
+       <li> * Vaccination as planned = Primary schedule + 2 booster doses </li>
+       <li> ** Seasonal vaccination campaigns: Autumn = 15 September to 15 December; Spring = 15 March to 15 June
```

Comment:
Here and below some terminal `</li>` statements are missing.

Also need to remove all the stray asterisks as they confuse the markdown